tests/general/pio_sync_tests.F90.in

```diff
@@ -356,7 +357,19 @@ PIO_TF_AUTO_TEST_SUB_BEGIN nc_mvar_msync_1d_bc
       call PIO_write_darray(pio_file, pio_vars(j), wr_iodesc, wbuf, ierr)
       PIO_TF_CHECK_ERR(ierr, "Failed to write darray : " // trim(filename))
       if(j == isync) then
+#ifdef PIO_TEST_CLOSE_OPEN_FOR_SYNC
+        call PIO_closefile(pio_file)
+
+        ierr = PIO_openfile(pio_tf_iosystem_, pio_file, iotypes(i), filename, PIO_write)
+        PIO_TF_CHECK_ERR(ierr, "Could not reopen file " // trim(filename))
+
+        do k=1,NVARS
+          ierr = PIO_inq_varid(pio_file, vnames(k), pio_vars(k))
+          PIO_TF_CHECK_ERR(ierr, "Could not inq var1 :" // trim(filename))
```

Comment:
var1 is not appropriate here. Should be "Could not inq a var" or appending the value of index k if possible.

Reply:
Thanks, will fix soon.